nemo/collections/asr/parts/submodules/cuda_graph_rnnt_greedy_decoding.py

```diff
@@ -293,6 +293,13 @@ def __call__(
         device: torch.device,
         partial_hypotheses: Optional[List[rnnt_utils.Hypothesis]] = None,
     ):
+        if x.device.type != "cuda":
```

Comment:
@galv I added this to automatically support non-cuda inputs when the CUDA graph decoder + frame-looping algorithm is enabled. If you think there is a better way, please let me know.

Reply:
Also, it is possible to avoid some tests for frame-looping algorithm instead of this solution.

Reply:
To be honest, if I were doing this, I would probably just throw an exception (what torch does if you pass a tensor on the wrong device to an op that runs on a different device). Well, I suppose the code already throws an exception and fails :P 

I'm just not a fan of "falling back" to something else when we cannot do what the user requests, but I know that decision has already been made a few places regarding the cuda graphs decoding work. So I think this is fine! Let me know if you need me to approve. Otherwise, I will let someone else more familiar with the rest approve.